elastic/kibana
# [Infrastructure UI] Add logs tab to hosts view flyout

## Summary

This PR adds the logs tab to the flyout and enables the LogStream flyout actions.

_Flyout_
<img width="1210" alt="image" src="https://github.com/elastic/kibana/assets/2767137/d5f93447-82cb-4f72-8b5e-988d32613138">


_Hosts View_
<img width="1210" alt="image" src="https://github.com/elastic/kibana/assets/2767137/ffff2e5b-9ce2-49a9-98a5-1bf6bc5dbb64">


### For reviewer
- The search bar is slightly broken. Sometimes, the results don't match with what's typed in the search bar. Since it's an existing problem in the logs component used in the flyout, we'll address this problem in another PR

### How to test this PR 

- Start a local Kibana instance
- Navigate to `Inventory > Hosts`
- Open the flyout for a host
  - Click on logs tab
  - Copy the URL with the logs tab open and open in a new tab (should keep the state)
  - Play with the search (It's a bit broken), copy the URL and open in a new tab (should keep the state)
